requirements-cwl.txt

```
@@ -1,4 +1,4 @@
 cwltool==3.1.20220210171524
 galaxy-tool-util
-ruamel.yaml==0.17.16
+ruamel.yaml==0.17.21
```

I think this should be a range; otherwise this has to happen in lockstep with `cwltool`
```suggestion
ruamel.yaml>=0.15,<=0.17.21
```
based upon `cwltool`'s `requirements.txt` for now (can increase the minimum version if desired)